metriken-exposition/src/parquet.rs

```diff
@@ -24,16 +27,23 @@ pub struct ParquetOptions {
 }
 
 impl ParquetOptions {
+    /// Create a new set of `ParquetOption` with the default values
     pub fn new() -> Self {
         Self::default()
     }
 
+    /// Sets the compression level for the parquet file. The default is no
+    /// compression. Set the compression level to a corresponding zstd level to
+    /// enable compression.
     pub fn compression_level(mut self, level: i32) -> Result<Self, ParquetError> {
         let compression = Compression::ZSTD(ZstdLevel::try_new(level)?);
         self.compression = compression;
         Ok(self)
     }
 
+    /// Sets the number of rows to be cache in memory before being written as a
+    /// `RecordBatch`. Large values have better performance at the cost of
+    /// additional memory usage. The default is ~1M rows (2^20).
```

Comment:
Not a blocker, but should we impose a check on the max size not being too large?

Reply:
I'm unsure about this and for now am opting to leave it without setting our own limit here. A reasonable upper bound varies depending on the system memory and row width. Some applications might have few metrics and wish to use multi-million row batches. Some may have extensive instrumentation and run OOM at even just a million rows.

For now, I'm going to defer on this